File: 351004/Nepesov/Lab1/untitled/src/main/java/org/example/dto/NewsResponseTo.java
package org.example.dto;

import com.fasterxml.jackson.annotation.JsonRootName;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import java.time.LocalDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class NewsResponseTo {
    private Long id;
    private Long editorId;
    private String title;
    private String content;
    private LocalDateTime created;
    private LocalDateTime modified;
}

File: 351003/brezgunov/distcomp/src/main/kotlin/com/distcomp/dto/news/NewsResponseTo.kt
package com.distcomp.dto.news

import com.distcomp.dto.marker.MarkerResponseTo
import com.distcomp.dto.notice.NoticeResponseTo
import com.distcomp.dto.user.UserResponseTo
import java.time.LocalDateTime

data class NewsResponseTo(
    val id: Long,
    val title: String,
    val content: String,
    val created: LocalDateTime,
    val modified: LocalDateTime,
    val user: UserResponseTo,
    val notices: MutableList<NoticeResponseTo>,
    val markers: MutableList<MarkerResponseTo>
)
